eventheader_macros/src/parser.rs

```diff
@@ -264,9 +292,9 @@ impl<'a> Parser<'a> {
     /// If skip_to_comma returns true, you may want to call comma_after_item.
     ///
     /// `while current() != ',' && current() != None { move_next(); }`
-    fn skip_to_comma(&mut self, next_token: TokenTree) -> bool {
+    fn skip_to_comma(&mut self, next_token: Option<TokenTree>) -> bool {
```

Comment:
Please don't do this. You added one place that might possibly pass None to this function, and it would be much better for that one place to just check for None itself.